### ISSUE TITLE
panic when calling getpage with page greater than total

### ISSUE THREAD

**Issue by jedy** [HouzuoGuo/tiedot#88]
run
```
curl "http://localhost:8080/create?col=Feeds"
curl --data-ascii doc='{"a": 1, "b": 2}' "http://localhost:8080/insert?col=Feeds"
curl "http://localhost:8080/getpage?col=Feeds&page=1&total=1"
```
and log:
```
2015/04/03 14:19:36 http: panic serving 127.0.0.1:65389: runtime error: slice bounds out of range
goroutine 6 [running]:
net/http.func·011()
	/usr/local/Cellar/go/1.4.1/libexec/src/net/http/server.go:1130 +0xbb
github.com/HouzuoGuo/tiedot/data.(*HashTable).collectEntries(0xc20803a220, 0x17ad3, 0xc209839480, 0x0, 0x10, 0xc209839500, 0x0, 0x10)
	/Users/jedy/Documents/work/go/src/github.com/HouzuoGuo/tiedot/data/hashtable.go:230 +0x48e
github.com/HouzuoGuo/tiedot/data.(*HashTable).GetPartition(0xc20803a220, 0x1, 0x1, 0xc208096000, 0x0, 0x100000, 0xc2088a6000, 0x0, 0x100000)
	/Users/jedy/Documents/work/go/src/github.com/HouzuoGuo/tiedot/data/hashtable.go:254 +0x165
github.com/HouzuoGuo/tiedot/data.(*Partition).ForEachDoc(0xc20803a1c0, 0x1, 0x1, 0xc20805dbc0, 0xc20800b151)
	/Users/jedy/Documents/work/go/src/github.com/HouzuoGuo/tiedot/data/partition.go:105 +0x51
github.com/HouzuoGuo/tiedot/db.(*Col).ForEachDocInPage(0xc20802e5a0, 0x1, 0x1, 0xc20805dbc0)
	/Users/jedy/Documents/work/go/src/github.com/HouzuoGuo/tiedot/db/col.go:227 +0xe1
github.com/HouzuoGuo/tiedot/httpapi.GetPage(0x565358, 0xc208030280, 0xc2080325b0)
	/Users/jedy/Documents/work/go/src/github.com/HouzuoGuo/tiedot/httpapi/document.go:111 +0x4be
```

Maybe you should check the parameters.

**Comment by HouzuoGuo** [comment]
Thanks very much, this will be resolved very soon.

**Comment by HouzuoGuo** [comment]
fixed in ef42065..40a7436
